src/sobesity/domain/views/skill.py

```diff
@@ -46,8 +49,14 @@ def get_skill(
 def create_skills(
     body: PostSkillBody, skill_service: ISkillService = Provide[Services.skill]
 ):
-    skill_service.batch_create(body.to_domain())
-    return Response(), 201
+    try:
+        skill_service.batch_create(body.to_domain())
+        return Response(), 201
+    except SkillNameUniqueViolation as e:
+        abort(403, e)
```

Comment:
Make the same return as above. Use like [here](https://github.com/vorobeybird/Sobesity-BE/blob/main/src/sobesity/domain/views/user.py#L44)